packages/core/src/graph/process-project-graph.ts

```diff
@@ -37,7 +37,8 @@ function visitProject(
   getDependantProjectsForNxProject(
     projectName,
     context.workspace,
-    ({ projectFile }, dependencyName) => {
+    ({ destProjectFile }, dependencyName) => {
```

Comment:
Change this to `(, dependencyName) => {`, to avoid unused variable declaration and fix the typescript build.